source/velocity_boundary_conditions/gplates.cc

```diff
@@ -659,26 +660,26 @@ namespace aspect
               // number current_file_number +/- 1, because current_file_number
               // is the file older than the current model time
               const int old_file_number =
-                (decreasing_file_order) ?
+                (decreasing_file_order)
+                ?
                 current_file_number - 1
                 :
                 current_file_number + 1;
 
               //Calculate new file_number
               current_file_number =
-                (decreasing_file_order) ?
-                first_data_file_number
-                - static_cast<unsigned int> (time_steps_since_start)
+                (decreasing_file_order)
+                ?
+                first_data_file_number - time_steps_since_start
                 :
-                first_data_file_number
-                + static_cast<unsigned int> (time_steps_since_start);
+                first_data_file_number + time_steps_since_start;
```

Comment:
can you make the order of operations between ? and +/- more clear with parentheses?